pkg/taskrun/description/taskrun_description.go

```diff
@@ -27,6 +41,10 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .TaskRun.Name }}
 {{- if ne $timeout "" }}
 {{decorate "bold" "Timeout"}}:	{{ .TaskRun.Spec.Timeout.Duration.String }}
 {{- end }}
+{{decorate "bold" "Labels"}}:
```

Comment:
Instead of showing the Labels mark here even if no labels are present, maybe it would make sense here to not show Labels at all if .PipelineRuns.Labels length is 0.